### ISSUE TITLE
Accessibility Audit: Touch Target Issues for various elements on Listings page

### ISSUE THREAD

**Issue by jnowell** [CityOfDetroit/bloom#964]
**ENVIRONMENT**

Mobile: OS | OS Ver | assistive technology | Browser**
iPhone Xr| 14.4.2 | Accessibility Inspector | Safari
Pixel 3a XL | Android 12 | Accessibility Scanner | Chrome

**ISSUE**

Ensure the Accessibility Inspector/Scanner are on, and go to the [Listings](https://sites-public-tj3gg4i5eq-uc.a.run.app/listings) page. A number of elements don't meet the minimum touch target size. The Minimum touch target size should be of 44x44 pts for iOS, and 48x48 dp Android. 

Elements below:

Menu button :The height is 24dp
Check your housing qualification : The height is 46dp
Filter listings: The height is 45dp
Detroit home connect : 168X18 pts
Apartment title : 280X27pts
